tensorflow/c/kernels/BUILD

```diff
@@ -24,6 +24,19 @@ tf_kernel_library(
     ],
 )
 
+tf_kernel_library(
+    name = "summary_op",
+    prefix = "summary_op",
+    deps = [
+        "//tensorflow/c:kernels",
```

Comment:
The correct ordering for this is:

        "//tensorflow/c:kernels",
        "//tensorflow/c:tf_status",
        "//tensorflow/c:tf_tensor",
        "//tensorflow/c/kernels:tensor_shape_utils",
        "//tensorflow/core:framework",
        "//tensorflow/core:lib",